src/autocomplete/EmojiProvider.tsx

```diff
@@ -85,7 +85,7 @@ export default class EmojiProvider extends AutocompleteProvider {
     public nameMatcher: QueryMatcher<ISortedEmoji>;
     private readonly recentlyUsed: IEmoji[];
     private emotes: Map<string, string> = new Map();
-    private emotesPromise: Promise<Map<string, string>>;
+    private emotesPromise!: Promise<Map<string, string>>;
```

Comment:
this doesn't seem right, the promise isn't set in a guaranteed place within the constructor

Reply:
switched to optional instead of non null assertion